apps/odyssey-stats/tsconfig.json

```diff
@@ -6,7 +6,12 @@
 		"typeRoots": [ "./node_modules/@types", "../../node_modules/@types" ],
 		"types": [ "node" ]
 	},
-	"include": [ "./src/**/*", "./package.json", "../../config/production.json" ],
+	"include": [
+		"./src/**/*",
+		"./package.json",
+		"../../config/production.json",
+		"../../client/my-sites/stats/components/stats-main/stats-main.tsx"
```

Comment:
Does this file mean [client/my-sites/stats/components/stats-main/index.tsx](https://github.com/Automattic/wp-calypso/pull/101205/files#diff-3292ca60fd671e1b7b1fbe050d29affd62dfa2e972408bb70c7a14201163bf0d) created from this PR?

Reply:
this is not necessary, removed